lib/routes/seagullApi.js

```diff
@@ -285,8 +285,20 @@ module.exports = function (crudHandler, userApiClient, gatekeeperClient, metrics
 
                 if (_.isEmpty(user.trustorPermissions)) {
                   delete user.profile.patient;
+                } else {
+                  if (user.trustorPermissions.view || user.trustorPermissions.upload) {
```

Comment:
You should add check for custodian here, too. Currently, all custodians have view and upload permission, too, but this might not always be the case.